docs/proposals/diagnose-experience.md

```diff
@@ -0,0 +1,247 @@
+# Improve ORAS diagnose experience
+
+ORAS currently offers two global options, `--verbose` and `--debug`, which enable users to generate verbose output and debug logs respectively. These features facilitate both users and developers in inspecting ORAS's performance, interactions with external services and internal systems, and in diagnosing issues by providing a clear picture of the tool’s operations.
```

Comment:
Why not call verbose log directly, is there any difference between verbose log and verbose ouput?